src/ert/run_models/base_run_model.py

```diff
@@ -315,17 +312,26 @@ def start_simulations_thread(
                     )
         except ErtRunError as e:
             self._completed_realizations_mask = []
-            self._failed = True
-            self._exception = e
+            failed = True
+            exception = e
         except UserWarning as e:
-            self._exception = e
+            exception = e
```

Comment:
This will never be shown I think :thinking: 
```suggestion
            pass
```